chartiq/development/js/standard.js

```diff
@@ -1127,14 +1127,26 @@ CIQ.ChartEngine.prototype.drawingClick = function (panel, x, y) {
 				this.adjustDrawings();
 				this.draw();
 				this.changeOccurred("vector");
-				this.controls.crossX.classList.replace(
-					"stx_crosshair_drawing",
-					"stx_crosshair"
-				);
-				this.controls.crossY.classList.replace(
-					"stx_crosshair_drawing",
-					"stx_crosshair"
-				);
+				if(typeof this.controls.crossX.classList === 'string'){
+					this.controls.crossX.classList.replace(
+						"stx_crosshair_drawing",
+						"stx_crosshair"
+					);
+				}else{
```

Comment:
seems like it needs extra spaces around `}else{` like `} else {`

Reply:
Fixed